lib/runner.nix

```
@@ -6,15 +6,16 @@
 let
   inherit (pkgs) lib;
 
-  inherit (microvmConfig) hostName;
+  inherit (microvmConfig) hostName vmHostPackages;
 
-  inherit (import ./. { inherit lib; }) createVolumesScript makeMacvtap withDriveLetters;
+  inherit (import ./. { inherit lib; }) makeMacvtap makeMacvtap withDriveLetters;
+  inherit (import ./volumes.nix { inherit lib microvmConfig; }) createVolumesScript;
   inherit (makeMacvtap {
     inherit microvmConfig hypervisorConfig;
   }) openMacvtapFds macvtapFds;
 
   hypervisorConfig = import (./runners + "/${microvmConfig.hypervisor}.nix") {
-    inherit pkgs microvmConfig macvtapFds withDriveLetters;
+    inherit microvmConfig macvtapFds pkgs;
```

```suggestion
    inherit pkgs microvmConfig macvtapFds withDriveLetters;
```
I think you did a bad merge here